scripts/install_pytorch.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
will this work on non-POSIX terminals ? as opposed to `#!/usr/bin/sh`?

Reply:
This invokes the `bash` shell explicitly (so it is not exactly POSIX compliant, or it might be but I haven't tested that), but shouldn't be an issue with either mac or linux which is what it caters to.